eZ/Publish/Core/Repository/User/PasswordHashService.php

```diff
@@ -9,10 +9,10 @@
 namespace eZ\Publish\Core\Repository\User;
 
 use eZ\Publish\API\Repository\Values\User\User;
-use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\Core\Repository\User\Exception\UnsupportedPasswordHashType;
 
 /**
- * @internal
+ * @throw UnsupportedPasswordHashType
```

Comment:
^ this seems to be unintended change (besides that it's incorrect)

Reply:
fixed